packages/dmn-js-drd/test/spec/features/drill-down/DrillDownSpec.js

```diff
@@ -252,6 +254,28 @@ describe('features - drilldown', function() {
       }
     ));
 
+    // FIXME:
+    xit('on Enter key pressed', inject(
+      function(eventBus, drillDown, elementRegistry) {
+
+        // given
+        var drillSpy = sinon.spy(drillDown, 'drillDown');
+
+        var overlayEl = queryOverlay('Decision_Table');
+        console.log(overlayEl.innerHTML);
+
+        // when
+        // press ENTER
+        triggerFocusIn(overlayEl);
+        triggerKeyEvent(overlayEl, 'keydown', 13);
+
+        // then
+        expect(drillSpy).to.have.been.calledOnce;
+
+        expect(drillSpy).to.have.been.calledWith(overlayEl);
```

Comment:
Working on fixing this test, at the moment overlayEl does return the button element, but the drillSpy is not called in test.